src/elements/radio-input/src/index.tsx

```diff
@@ -5,66 +5,27 @@ import { jsx } from '@emotion/core'
 
 import * as st from './styles'
 
-type Value = string | number | boolean
-
-export interface DataProps {
-  [key: string]: boolean | number | string | null
-}
-
-interface Props {
-  dataProps?: DataProps
+interface Props extends React.InputHTMLAttributes<HTMLInputElement> {
   label: string
   name: string
-  onBlur: () => void
-  onChange: (value: Value) => void
-  onFocus?: () => void
-  selected: Value
   width?: Width
-  value: Value
 }
 
-const prependDataProps = (dataProps: DataProps) =>
-  Object.keys(dataProps).reduce(
-    (props, key) => ({
-      ...props,
-      [`data-${key}`]: dataProps[key]
-    }),
-    {}
-  )
-
 export type Width = 'half' | 'full'
 
 export const RadioInput: React.FC<Props> = ({
-  dataProps = {},
   label,
-  name,
-  onBlur,
   onChange,
```

Comment:
Shouldn't `label` and `onChange` be passed down?

Reply:
`onChange` yes! gets a bit tricky to test onChange in storybook.
`label` makes sense to extract as it's only used in the span and not a property of input.

Reply:
> gets a bit tricky to test onChange in storybook
Apparently I'm wrong, I'll add this